github/ci/prow/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -67,3 +67,32 @@ presubmits:
               memory: "1Gi"
             limits:
               memory: "1Gi"
+  - name: kubernetes-crud-ci-role
+    always_run: false
+    run_if_changed: "github/ci/kubernetes-crud/.*"
+    decorate: true
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror: "true"
+      preset-kubevirtci-docker-credential: "true"
+    spec:
+      nodeSelector:
+        type: bare-metal-external
+      containers:
+        - image: gabrielecerami/kubevirt-ci-testenv
+          # docker run \
+          # -v /project-infra:/opt/source/project-infra
+          # -v /dev/kvm:/dev/kvm
+          command:
+            - "/usr/local/bin/runner.sh"
+            - "/bin/bash"
+            - "-c"
+            - "/etc/init.d/docker start; source /opt/virtualenv/bin/activate && cd github/ci/kubernetes_crud/ &&  molecule test"
```

Comment:
Why is it necessary to start the docker daemon manually?

Reply:
In some of my tests the docker daemon was not started with the container. But I changed a few things since. For example the docker file doesn't start anymore on an old version of bootstrap. I can add an additional test removing it, and see if it's really needed.

Reply:
Job succeded, removing